test/ModuleManifest.Tests.ps1

```diff
@@ -1,3 +1,4 @@
+[System.Diagnostics.CodeAnalysis.SuppressMessage('PSUseDeclaredVarsMoreThanAssigments', '', Scope='*', Target='SuppressImportModule')]
```

Comment:
That seems reasonable, It wasn't a VSCode warning, I just got a bit carried away suppressing warnings that came up.

How would we do that? Just ignore analysis results from `*.Tests.ps1` files, or is there another approach.